cypress/common/youth-pass-constants.js

```diff
@@ -0,0 +1,7 @@
+const getYouthPassConstants = () => {
+    return {
+        youthPassUrl: "https://mbta.preprod.simpligov.com/preprod/portal/ShowWorkFlow/AnonymousEmbed/86cadfa6-e8ea-46f1-b6e8-62498f066962"
```

Comment:
I think this will need to be read from an OS-level environment variable in order for CI to be able to set a value. Looking at [this documentation page](https://docs.cypress.io/guides/guides/environment-variables) super quickly, I wonder if we set the pre-prod URL as the default using [Option #1](https://docs.cypress.io/guides/guides/environment-variables#Option-1-configuration-file) and then use [Option #3](https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_) (or if that doesn't work for some reason maybe [Option #4](https://docs.cypress.io/guides/guides/environment-variables#Option-4-env)) to overwrite it in CI or when needed by developers. I'm open to other approaches though.

Reply:
Makes sense! I've incorporated Option 1 into `cypress.json` and removed this constants file